README.md

```diff
@@ -8,4 +22,47 @@ SLACK_APP_TOKEN=
 REPORT_CHANNEL=bot-test
 ```
 
-We use 'Socket mode' so no need to proxy Slack's requests
+We use 'Socket mode' so no need to proxy Slack's requests.
+
+Rename "env.template.txt" to ".env" which is gitignored and safe for secrets.
+You can source into your shell with:
+
+ ```bash
+$ set -o allexport; source .env; set +o allexport
+ ```
+
+#### Running locally
+
+Install dependencies by executing the following command:
+
+ ```bash
+$ npm install
+ ```
+Run:
+
+```bash
+$ node app.js
+```
+
+The applications's health page will be available at https://localhost:3000/health
+
```

Comment:
not really relevant I think it's just used to satisfy kubernetes that it's running we don't add any info there
```suggestion
```